docs/extensions/twitter.md

```diff
@@ -0,0 +1,36 @@
+## Twitter to Roam Integration
+
+The Twitter integration allows users to import all their tweets pertaining to a particular page. So, if you've tweeted about `Trump` on twitter, you can head over to the `Trump` page on roam, and then pull all your tweets on Trump.
```

Comment:
could we use a less polarizing example lol. don't actually care too much

Reply:
Haha. Changed it :)